examples/article-with-docked-video.html

```diff
@@ -0,0 +1,596 @@
+      padding-right: 20px;
+    }
+
+    .jump-to {
+      padding-left: 15px;
+      padding-right: 15px;
+    }
+
+    .jump-to ul a, .jump-to ol a {
+        text-decoration: none;
+        color: #2294dd;
+    }
+
+    #top-bar {
+      background: white;
+      border-bottom: 1px solid #e2e2e2;
+      position: fixed;
+      right: 0;
+      left: 0;
+      z-index: 10000000000;
+    }
+
+    header#article-desc {
+      background: rgb(250, 250, 250);
+      padding: 20px;
+    }
+
+    header#article-desc h1 {
+      margin-top: 0px;
+      padding-top: 0px;
+    }
+
+    .slot-fallback {
+       padding: 16px;
+       background: #1b1b1b;
+       color: white;
+     }
+
+     #lightbox-close-btn {
+       font-family: Helvetica, sans-serif;
+       color: white;
+       position: absolute;
+       top: 0px;
+       right: 0px;
+       padding: 20px;
+     }
+
+     main {
+      max-width: 800px;
+      margin: 0 auto;
+      padding-top: 90px;
+     }
+
+     amp-video {
+      background: rgba(210, 210, 210, 1);
+     }
+
+     #more-articles {
+       background: #fafafa;
+       padding: 20px 20px 0;
+     }
+     #more-articles ul {
+      margin: 0;
+      padding: 0;
+      list-style: none;
+     }
+     #more-articles li {
+      border-top: 1px solid #f0f0f0;
+      padding: 10px 20px;
+      margin: 0 -20px;
+     }
+     #more-articles h3 {
+      margin: 0 0 20px;
+     }
+    @media screen and (max-width: 600px) {
+      main {
+        padding-right: 0;
+      }
+      aside {
+        display: none;
+      }
+      #top-bar {
+        position: static;
+      }
+    }
+  </style>
+  <script async custom-element="amp-animation" src="https://cdn.ampproject.org/v0/amp-animation-0.1.js"></script>
+  <script async custom-element="amp-sidebar" src="https://cdn.ampproject.org/v0/amp-sidebar-0.1.js"></script>
+  <script async custom-element="amp-video" src="https://cdn.ampproject.org/v0/amp-video-0.1.js"></script>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <meta name="apple-itunes-app" content="app-id=828256236, app-argument=medium://p/cb7f223fad86">
+  <link rel="manifest" href="medium-manifest.json">
+</head>
+<body>
+  <aside>
+    <amp-video-dock-slot layout="responsive" width="16" height="9" id="dock-slot">
```

Comment:
Anything special about this component that a normal `div` or `amp-layout` won't be able to handle? Maybe `dock=slotId` or `dock-slot=id`` can handle that? So far we have avoided tags that don't have an implementation component behind them.